pkg/cache/cache.go

```diff
@@ -203,6 +209,7 @@ func convertToSelectorsByGVK(selectorsByObject SelectorsByObject, scheme *runtim
 		}
 		selectorsByGVK[gvk] = internal.Selector(selector)
 	}
+	selectorsByGVK[schema.GroupVersionKind{}] = internal.Selector(defaultSelector)
```

Comment:
nit not blocking 
const defalutSelectorKey = schema.GroupVersionKind{}

Reply:
Needs to be a var but good idea, updated the code.